vendor/github.com/gorilla/mux/regexp.go

```diff
@@ -296,7 +296,7 @@ func (v *routeRegexpGroup) setMatch(req *http.Request, m *RouteMatch, r *Route)
 					} else {
 						u.Path += "/"
 					}
-					m.Handler = http.RedirectHandler(u.String(), http.StatusMovedPermanently)
+					m.Handler = http.RedirectHandler(u.String(), http.StatusPermanentRedirect)
```

Comment:
We probably shouldn't change the vendored code like this. Poking around the docs it seems like we may need to change the handling for this in the OPA server code

Reply:
Oh, right - I forgot that golang loves vendoring. I'll see if I can trace this.